src/components/externalLinks/ExternalLinks.component.tsx

```diff
@@ -197,6 +197,54 @@ const ExternalLinkIframeModal = ({ selectedExternalLink, handleCloseModal }) =>
     </VisibleModal2>
 )
 
+const ExternalLinkChip = ({ linkOption, idx, handleOpenModal, details, isOverviewPage }: ExternalLinkChipProps) => {
+    const externalLinkURL = getParsedURL(true, linkOption.value.toString(), details)
+    const handleTextClick = () => handleOpenModal(linkOption, externalLinkURL)
+    return (
+        <ConditionalWrap
+            key={`${linkOption.label}-${idx}`}
+            condition={!!linkOption.description}
+            wrap={(children) => (
+                <TippyCustomized
```

Comment:
This will break in eslint since we should not define component in prop as method